src/views/HomeRedesign/HomeSection.tsx

```diff
@@ -0,0 +1,68 @@
+import { Heading } from "@chakra-ui/react";
+import { Fragment, FunctionComponent, useMemo } from "react";
+import { CatalogPackage } from "../../api/package/packages";
+import { PackageList } from "../../components/PackageList";
+import { useCatalog } from "../../contexts/Catalog";
+
+export interface HomeSectionProps {
+  name: string;
+  showLastUpdated?: number;
+  showPackages?: string[];
+}
+
+export const HomeSection: FunctionComponent<HomeSectionProps> = ({
+  name,
+  showLastUpdated,
+  showPackages,
+}) => {
+  const { data, loading, error } = useCatalog();
+
+  const results = useMemo(() => {
+    if (loading || error || !data?.packages) return [];
+
+    let packages = data.packages;
+
+    if (showLastUpdated) {
+      // sort packages by date
+      packages = [...packages]
+        .sort((p1, p2) => {
+          const d1 = new Date(p1.metadata.date);
```

Comment:
Could we use the `SORT_FUNCTIONS[CatalogSearchSort.PublishDateDesc]` helper that `CatalogSearchAPI` uses in place of this? I realize the code is identical, but it's an easy opportunity to be a little DRYer